### PR TITLE
Add hillbilly profession

### PR DESCRIPTION
**Pull request by Froffy025**
#### Summary
Content "Adds hillbilly profession."

#### Purpose of change
Adds a hillbilly profession with an extended 4-seater pickup truck, a shotgun, a 6 pack of beers, and a box of shotgun shells. Yee-haw. 

#### Testing
Made sure I got the whole six-pack of beer.
Made sure the truck worked. 
Made sure the game didn't break.

### CODE REVIEW

**Comment by ampersand55** [comment]
How about giving them a banjo? Or is that too on the nose?

**Comment by MNG-cataclysm** [comment]
> How about giving them a banjo? Or is that too on the nose?

I think that’s too on-the-nose, personally.

**Comment by Froffy025** [comment]
> How about giving them a banjo? Or is that too on the nose?

Starting mission to find a musical instrument after you smashed your poor banjo over your zombie cousin's head LMAO

File: data/json/professions.json [hunk 1]
```diff
@@ -35,6 +35,19 @@
       { "item": "p320mag_17rd_9x19mm", "ammo-item": "9mmfmj", "charges": 17 }
     ]
   },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "six_pack_beer",
+    "entries": [
+      { "item": "beer" },
```
Comment (casswedson) [inline data/json/professions.json:43]: I would just add this to the profession items like `{ "item": "beer", "count": 6 }` much cleaner

also make sure all those beers fit in the character's inventory, add a sling pack or something that fits the theme

**Review by MNG-cataclysm — APPROVED** [review]

**Comment by Maleclypse** [comment]
In New England they are referred to as Yanknecks.

**Comment by Froffy025** [comment]
> In New England they are referred to as Yanknecks. iirc

Hillbilly feels more inclusive and easier for anyone to recognize to me, but this is coming from a person from the South. "Redneck" seems to imply you need to be a certain race, and I've never heard anyone use the word "yankneck", but I feel it implies that you have to come from New England specifically.

**Comment by Maleclypse** [comment]
> > In New England they are referred to as Yanknecks. iirc
> 
> Hillbilly feels more inclusive and easier for anyone to recognize to me, but this is coming from a person from the South. "Redneck" seems to imply you need to be a certain race, and I've never heard anyone use the word "yankneck", but I feel it implies that you have to come from New England specifically.

I thought more on it and remembered it's from this book https://www.amazon.com/Druids-Cherohonkees-Creatures-Unique-Republic/dp/0452285623#:~:text=Book%20details&text=A%20hilarious%20take%20on%20the,who%20dress%20like%20Native%20Americans. That I read long before I moved out of the south and then back again. Technically it covered anyone with a confederate flag north of the mason dixon line.  So basically everything from Maryland up.

**Pull request merged**